webapp/messages/en.json

```
@@ -172,10 +172,21 @@
     "hemidocs": "Hemi Docs",
     "network-status": "Network Status",
     "swap": "Swap",
+    "stake": "Stake",
     "tools": "Tools",
     "tunnel": "Tunnel",
     "web-pop-miner": "Web PoP Miner"
   },
+  "stake-page": {
+    "dashboard": {
+      "title": "Dashboard",
+      "subtitle": "Stake your assets to earn rewards and boost your points."
```

```suggestion
      "subtitle": "Stake your assets to earn rewards and boost your points."
      "title": "Dashboard",
```

Let's sort by key (yes, this should be automated 😓 )

Same on line 186-187 and in Spanish files

---

Fixed.